source/grid/grid_in.cc

```diff
@@ -1497,20 +1556,19 @@ GridIn<dim, spacedim>::read_msh(std::istream &in)
     unsigned int global_vertex = 0;
     for (int entity_block = 0; entity_block < n_entity_blocks; ++entity_block)
       {
-        int           tagEntity;
-        int           dimEntity;
         int           parametric;
         unsigned long numNodes;
```

Comment:
I didn't realize the camel case code was already here. We should just deal with that in a followup PR.